backend/functions/lib/use-cases/user-session/user-session.ts

```diff
@@ -85,16 +85,19 @@ export class UserSessionUseCase {
       await sessionCacheRepository.put(context, session);
 
       return session;
-    } catch (originalError) {
-      if (isConflictError(originalError)) {
+    } catch (error) {
+      const isConflict = error.originalError
+        ? isConflictError(error.originalError)
+        : isConflictError(error);
```

Comment:
Some time back we [wrapped up errors](https://github.com/US-Trustee-Program/Bankruptcy-Oversight-Support-Systems/commit/fe6b6da4b27b79fe440bece9f720585268616e1f#diff-81ae118bcc7d49a117924c2b30160411a03c88597b4df541f7e47897a6c63089L38) in the `execute` function on CosmosDbRepository. So here, the call to `isConflictError` was returning false.